go.mod

```diff
@@ -2,61 +2,69 @@ module github.com/keycloak/keycloak-operator
 
 require (
 	cloud.google.com/go v0.46.3 // indirect
-	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/bombsimon/wsl v1.2.8 // indirect
 	github.com/coreos/prometheus-operator v0.29.0
-	github.com/go-openapi/spec v0.19.0
-	github.com/gogo/protobuf v1.3.0 // indirect
+	github.com/go-critic/go-critic v0.4.0 // indirect
+	github.com/go-openapi/spec v0.19.2
```

Comment:
It seems you ran `go mod tidy` or `go mod vendor`. Could you please put vendoring changes into a separate commit? This way, reviewing will be much easier.